tests/compute/test_transform.py

```diff
@@ -2350,6 +2350,58 @@ def test_module_laplacian_pe(idtype):
     else:
         assert F.allclose(new_g.ndata['lappe'].abs(), tgt)
 
+@parametrize_dtype
+def test_module_normalize_features(idtype):
+    transform = dgl.NormalizeFeatures()
+
+    # Case1: Normalize features of a homogenous graph.
+    g = dgl.rand_graph(5, 10, idtype=idtype, device=F.ctx())
+    g.ndata['h'] = F.randn((g.num_nodes(), 3))
+    g.edata['w'] = F.randn((g.num_edges(), 2))
+    new_g = transform(g)
+    assert F.allclose(new_g.ndata['h'].sum(1), F.tensor([1.0, 1.0, 1.0]))
+    assert F.allclose(new_g.edata['w'].sum(1), F.tensor([1.0, 1.0]))
+
+    # Case2: Normalize features of a heterogeneous graph.
+    g = dgl.heterograph({
+        ('user', 'follows', 'user'): (F.tensor([1, 2]), F.tensor([3, 4])),
+        ('player', 'plays', 'game'): (F.tensor([2, 2]), F.tensor([1, 1]))
+    })
+    g.ndata['h'] = {'game': F.randn(2, 5), 'player': F.randn(3, 5)}
```

Comment:
Add a feature `h2` for nodes of type `user`?

Reply:
Added